content/en/docs/ambient/upgrade/helm-upgrade/index.md

```diff
@@ -4,96 +4,103 @@ description: Upgrading an ambient mode installation with Helm.
+    $ kubectl apply -f manifests/charts/base/crds
     {{< /text >}}
 
-### Upgrade the Istio discovery Component
+## Install the new Control Plane
 
-Istiod is the control plane component that manages and configures the proxies to route traffic within an ambient mesh.
+Istiod is the control plane component that manages and configures the proxies to route traffic within an ambient mesh. The following command will install a new instance of this control plane alongside the old, but will not introduce any new proxies, or take over control for existing proxies. If you have previously customized your istiod installation, you can reuse the `values.yaml` file from previous upgrades or installs to keep your control planes consistent.
 
 {{< text syntax=bash snip_id=upgrade_istiod >}}
-$ helm upgrade istiod istio/istiod -n istio-system
+$ helm install istiod-"$REVISION" istio/istiod -n istio-system --set revision="$REVISION" --set profile=ambient --wait
 {{< /text >}}
 
-### Upgrade the ztunnel component
+## Upgrade the ztunnel DaemonSet
 
-The ztunnel DaemonSet is the node proxy component.
-
-{{< warning >}}
-As ambient mode is not yet Stable, the following statement is not a compatibility guarantee, and is subject to change, or removal. Prior to reaching Stable status, this component and/or the control plane may receive breaking changes that prevent compatibility between minor versions.
-{{< /warning >}}
-
-The ztunnel at version 1.x is generally compatible with the control plane at version 1.x+1 and 1.x. This means the control plane must be upgraded before ztunnel, as long as their version difference is within one minor version.
+The {{< gloss >}}ztunnel{{< /gloss >}} DaemonSet is the node proxy component. The ztunnel at version 1.x is compatible with the control plane at version 1.x+1 and 1.x. This means the control plane must be upgraded before ztunnel, as long as their version difference is within one minor version. If you have previously customized your ztunnel installation, you can reuse the `values.yaml` file from previous upgrades or installs to keep your {{< gloss >}}data plane{{< /gloss >}} consistent.
 
 {{< warning >}}
 Upgrading ztunnel in-place will briefly disrupt all ambient mesh traffic on the node.
 Node cordoning and blue/green node pools are recommended to mitigate blast radius risk during production upgrades. See your Kubernetes provider documentation for details.
 {{< /warning >}}
 
 {{< text syntax=bash snip_id=upgrade_ztunnel >}}
-$ helm upgrade ztunnel istio/ztunnel -n istio-system
+$ helm upgrade ztunnel istio/ztunnel -n istio-system --set revision="$REVISION" --wait
 {{< /text >}}
 
-### Upgrade the CNI component
+## Upgrade the CNI DaemonSet
 
 The Istio CNI agent is responsible for detecting pods added to the ambient mesh, informing ztunnel that proxy ports should be established within added pods, and configuring traffic redirection within the pod network namespace. It is not part of the data plane or control plane.
```

Comment:
This is not your edit, but surely it's part of the control plane?

Reply:
I can't tell from your comment what the 'it' is that you are asking about.

Reply:
"The CNI component"... "is not part of the data plane or control plane"